dts/bindings/clock/atmel,sam0-fdpll.yaml

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2019 Derek Hageman <hageman@inthat.cloud>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+title: SAM0 Fractional Digital Phase Locked Loop (FDPLL)
+version: 0.1
+
+description: >
+    This is a representation of a SAM0 Fractional Digital Phase Locked Loop.
+    This is a programmable multiplier on an input reference clock.
+
+inherits:
+    !include clock.yaml
+
+properties:
+    compatible:
+      constraint: "atmel,sam0-fdpll"
+
+    clock-frequency:
```

Comment:
What is `clock-frequency` suppose to mean?

Reply:
This is the output frequency of the clock provider.  It is used together with the input clock frequency and the electrical limits of the device to calculate the divisor(s) and the multiplier required to produce the frequency specified. 

I have clarified the description on the binding.